### ISSUE TITLE
Add a changelog

### ISSUE THREAD

**Issue by soullivaneuh** [Soullivaneuh/kodi-plugin.audio.napster#1]
Add a changelog when first stable version will be released.

According to [dev guide](https://xbmc-gpodder-integration.googlecode.com/files/XBMC%20Addon%20Developers%20Guide%20-R7.pdf):

> **changelog.txt**
> 
> A text file that contains a description of each release change that you make to the system. This is
> displayed in the XBMC addon installation/update system. Recommended format is to have it
> sorted by version in descending order, with a simple description as to the major changes (new
> functionality, big fixes etc,) in each version. (In the author's opinion, too many addons skip this
> piece of information making it difficult for users to determine whether a particular problem that
> they may have been having has been fixed or not.)

**Comment by SLiX69** [comment]
Hi, any update regarding this addon?

**Comment by soullivaneuh** [comment]
@SLiX69 Not really. No available API for Napster actually, at least for France... :-/
